src/chai-form.ts

```diff
@@ -368,6 +371,8 @@ export class ChaiForm extends LitElement {
           <chai-phone></chai-phone>
           <chai-email></chai-email>
           <chai-address></chai-address>
+          <chai-destination></chai-destination>
+          <chai-move-size></chai-move-size>
```

Comment:
Are we sure that we want these two fields to be added to *all* default-formatted sites? Just checking. Maybe it's worth doing a progressive rollout here?

Reply:
There really is no reason to push these out all at once. I have removed them and we should do a progressive rollout.